data/json/monsters/turrets.json

```diff
@@ -331,5 +331,52 @@
     "death_function": [ "BROKEN" ],
     "death_drops": { "groups": [ [ "turret_speaker", 1 ] ] },
     "flags": [ "SEES", "NOHEAD", "ELECTRONIC", "IMMOBILE", "NO_BREATHE" ]
+  },
+  {
+    "id": "mon_exodii_turret",
+    "type": "MONSTER",
+    "name": "upcycled turret",
+    "description": "This hefty turret appears to be bolted together out of various scraps of technology, many of them extremely foreign looking.  It is equipped with a hefty looking machine gun.",
+    "default_faction": "exodii",
+    "weight": "206 kg",
+    "volume": "105 L",
+    "hp": 240,
+    "speed": 100,
+    "material": [ "steel" ],
+    "symbol": "2",
+    "color": "light_gray",
+    "aggression": -10,
+    "morale": 100,
+    "armor_bash": 24,
+    "armor_cut": 26,
+    "vision_day": 50,
+    "vision_night": 50,
```

Comment:
Having perfect night vision without an aiding searchlight is probably overpowered(applies to all exodii, but this one esp. since every other turret has a NV range of 3).